source/lib/Serialize.cpp

```diff
@@ -478,7 +478,7 @@ populateParentNamespaces(
             InfoType::IT_namespace);
 }
 
-void PopulateTemplateParameters(std::optional<TemplateInfo>& TemplateInfo,
+void PopulateTemplateParameters(llvm::Optional<TemplateInfo>& TemplateInfo,
```

Comment:
Why don't we just replace it everywhere?

Reply:
I did.